hwy/ops/x86_avx3-inl.h

```diff
@@ -487,6 +495,8 @@ HWY_API size_t CompressBitsStore(VFromD<D> v, const uint8_t* HWY_RESTRICT bits,
   return CompressStore(v, LoadMaskBits(d, bits), d, unaligned);
 }
 
+#pragma pop_macro("HWY_X86_SLOW_COMPRESS_STORE")
```

Comment:
Can you help me understand why we pop_macro? It seems `HWY_X86_SLOW_COMPRESS_STORE` is only defined in this file.

Reply:
Oh, I see, it depends on the target and we want to reset it for each target.